ui/pages/create-account/new-account.component.js

```diff
@@ -43,21 +50,39 @@ export default class NewAccountCreateForm extends Component {
         });
     };
 
+    const accountNameExists = (allAccounts, accountName) => {
+      const accountsNames = allAccounts.map((item) => item.name);
+
+      return accountsNames.includes(accountName);
+    };
+
     return (
       <div className="new-account-create-form">
         <div className="new-account-create-form__input-label">
           {this.context.t('accountName')}
         </div>
         <div>
           <input
-            className="new-account-create-form__input"
+            className={classnames('new-account-create-form__input', {
+              'new-account-create-form__input__error': accountNameExists(
+                accounts,
+                newAccountName,
+              ),
+            })}
             value={newAccountName}
             placeholder={defaultAccountName}
             onChange={(event) =>
               this.setState({ newAccountName: event.target.value })
             }
             autoFocus
           />
+          {accountNameExists(accounts, newAccountName) ? (
+            <div
+              className={classnames('send-v2__error', 'send-v2__error-amount')}
```

Comment:
These classes were written for the "Send" page, and should not be used here. The styles for this component should be under the `new-account-create-form` namespace.

Reply:
Fixed in: d69e6682ef8cce5a6f0d7dd860acabac7ec614e3